src/progress_bar.jl

```diff
@@ -29,7 +29,7 @@ function next!(p::ProgressBar, io::IO = stdout)
     previous_time = p.previous_time
     interval = p.interval
 
-    (time() - previous_time) < interval && return
+    ((time() - previous_time) < interval || (time() - start_time) < interval) && return
```

Comment:
should we add `return nothing` here?
To make the code more understandable

Reply:
and also in line 19, line 23

Reply:
What does `flush` return? We should return the same type. If it returns nothing, yes, we can also return nothing. However, using just `return` is often used in C, so it is not something new.